.prettierrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
What if:
```
const eslintConfig = require("@react-hookz/eslint-config/.prettierrc.js")

module.exports = {
  ...eslintConfig,
};
```

This way it inherits the config used in eslint and easily can be extended

Reply:
Good idea! I can also put it into `package.json` like you said on Discord, but the config file seems to follow what's been done with other similar tools.

Let me know!

Reply:
i'd stay with separate file.